misc/pytorch_toolkit/miriad_compression_codes/src/utils/dataloader.py

```diff
@@ -0,0 +1,142 @@
+from torch.utils import data
+from PIL import Image
+import os
+import random
+import torch
+import pandas as pd
+
+
+class CustomDatasetPhase1(data.Dataset):
+	
+    def __init__(self, path_to_dataset, files256=None,files128=None,split=None,
+				transform_images = None, transform_masks = None,
+				images_path_rel = '.', masks_path_rel = '.',
+				preserve_names = False):
+        self.path_to_dataset = os.path.abspath(path_to_dataset) # root folder of the CBIS-DDSM dataset
```

Comment:
remove the word CBIS-DDSM from the comments.

Reply:
removed